src/search/views/Search.tsx

```diff
@@ -250,7 +252,7 @@ const Search: FunctionComponent<SearchProps & RouteComponentProps> = ({
 		console.log('TODO handle search result bookmark button toggle', active, id);
 	};
 
-	const handleOriginalCpLinkClicked = async (id: string, originalCp: string | undefined) => {
+	const handleOriginalCpLinkClicked = async (originalCp: string | undefined) => {
```

Comment:
ik denk dat ik hier id nog meegaf om in de toekomst via de gestandaardiseerde naam te zoeken ipv de naam die opgeslagen is bij het media item.